docs/UserGuide.md

```diff
@@ -64,16 +78,25 @@ Views a module stored in the system
   Examples:
    * `viewmodule n/cs2103t` views the specified module
 
-#### Adding a zoom link to a module: `add zoom`
-
-  Adds a zoom link to an existing module.
+#### Adding a zoom link for a specific lesson to a module: `addzoom`
 
-  Format: `add zoom` **_`[MODULE_NAME]`_** **_`[ZOOM_LINK]`_**
+  Adds a zoom link for a specific lesson to an existing module.
 
-  * Adds a zoom link [ZOOM_LINK] to a module named **_`[MODULE_NAME]`_**
-
-  Example of usage:
-  `add zoom cs2103T https://sample.zoom.us` adds a zoom link `https://sample.zoom.us` to the module named `cs2103T`
+  Format: `addzoom INDEX n/LESSON_NAME z/ZOOM_LINK`
+  
+   * Adds a zoom link to the module at the specified `INDEX`. 
+   
+   * The index refers to the index number of the module shown on the displayed module list. 
+   
+   * The index **must be a positive integer** 1, 2, 3...
+   
```

Comment:
If I remember correctly (please correct me if I'm wrong hahaha), I think Dr. Wee mentioned about using a more friendly-term instead of using "integer"? Maybe we can consider using a more general term like "positive round number" (not sure if this defines the integer correctly though 😅 )? But I also think that integer should be fine since the target users are computing students that most likely know the term "integer".

Reply:
Yes, I think integer is fine in the context of our target audience.